### PR TITLE
Created a stub file for the UK

### LINKED ISSUE #44: username-is-already-taken2 recommendations from UK
# UK

- [x] Create a branch
- [x] Add a file
- [x] Commit my changes
- [x] Open a pull request
- [ ] Collaborate with my new friends
- [ ] Get a :+1: 
- [ ] Merge a pull request with master

### PR DESCRIPTION
**Pull request by username-is-already-taken2**
This closes #44

### CODE REVIEW

File: username-is-already-taken2-uk.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## UK
+
+### Places to eat
```
Comment (username-is-already-taken2) [inline username-is-already-taken2-uk.md]: I think this should be a different case

Comment (rkrajews) [inline username-is-already-taken2-uk.md]: agree; thanks for the info;

**Comment by sirance** [comment]
@username-is-already-taken2 What fun things can you do in Bracknell?? :smiley:

**Comment by username-is-already-taken2** [comment]
Looks goto :shipit:

**Pull request merged**